pygradle-plugin/src/test/groovy/com/linkedin/gradle/python/util/PackageInfoTest.groovy

```diff
@@ -75,6 +75,14 @@ class PackageInfoTest extends Specification {
         thrown(GradleException)
     }
 
+    def 'can parse a wheel distribution'() {
+        when:
+        def packageInfo = packageInGradleCache('foo-1.0.0-py2-none-any.whl')
+        then:
+        assert packageInfo.name == 'foo'
+        assert packageInfo.version == '1.0.0-py2-none-any'
```

Comment:
The version is not extracted correctly.
It should be 1.0.0 only.
This would break later when we try to install `foo==1.0.0-py2-none-any` which wouldn't exist. Pip would require` foo==1.0.0` there.
If we want to enable wheels to be handled through this same interface, we'd have to change the approach here and use a different matching.
This was clearly intended for `sdist` only.
There's a code in pygradle that parses wheels and finds the name, version, ... correctly off them.
Perhaps you could look into using that API.

Reply:
Agreed. I updated the pr accordingly.